### PR TITLE
beets-204-database-fixes

### PR DESCRIPTION
**Pull request by levertyk**
### Definition of Done:

* [x] Have you checked your [Jira](https://bee-cs458.atlassian.net/jira/software/projects/BEETS/boards/2/backlog) task, to make sure the user acceptance criteria are met?
* [x] Does your code have comments describing it's function and reason?

### Tests

* [ ] Do you have Jest tests?
No, Jest testing has not yet been implemented
* [x] Have you documented your testing criteria and edge cases in your Jira sub-task?

### Source Control

* [x] Is all relevant code checked-in?
* [x] Is this branch ready to be reviewed by the Tech Lead?

### Formatting

* [x] Do your variable names make sense (to others)?
* [x] Is your code style consistent with other code in the file or the [Style Guide](https://google.github.io/styleguide/jsguide.html)

### CODE REVIEW

File: utilities/DatabaseUtilities.js [hunk 1]
```diff
@@ -1,92 +1,101 @@
...
+import mysql from "mysql2/promise";
+import fs from "fs";
+import path from "path";
 
 let pool = null;
 
 export async function createSchema() {
-    const schemaPath = path.resolve('schema.sql');
-    if (!fs.existsSync(schemaPath)) throw new Error("Schema file not found!");
+  const schemaPath = path.resolve("schema.sql");
```
Comment (piratecody) [inline utilities/DatabaseUtilities.js:8]: I appreciate the use of double quotes instead of single quotes.

Double quotes GOAT

**Review by piratecody — APPROVED** [review]

File: utilities/DatabaseUtilities.js (hunk 1 above)
Comment (levertyk) [reply utilities/DatabaseUtilities.js:8]: Preach it, brother. Amen.
